src/classes/Model/Users/UpdateToken.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace dhope0000\LXDClient\Model\Users;
+
+use dhope0000\LXDClient\Model\Database\Database;
+
+class UpdateToken
+{
+    public function __construct(Database $database)
+    {
+        $this->database = $database->dbObject;
+    }
+
+    public function update(int $userId)
```

Comment:
Can this be renamed to `invalidateToken` it would probably be better just to delete the row to prevent developer errors like not checking the used column :smile: